daemon/daemon_windows.go

```diff
@@ -485,6 +489,12 @@ func (daemon *Daemon) setDefaultIsolation() error {
 			if containertypes.Isolation(val).IsHyperV() {
 				daemon.defaultIsolation = containertypes.Isolation("hyperv")
 			}
+			if containertypes.Isolation(val).IsProcess() {
```

Comment:
Shouldn't this be moved to `IsValid()`? IIUC for this type of daemon, "process" isolation simply isn't a valid option